resources/event-hackathon/accessibility-tips-for-jupyter-notebooks.md

```diff
@@ -0,0 +1,94 @@
+# Accessibility Tips for Creating & Publishing Jupyter Notebooks
+
+By the [Notebooks for all](https://github.com/Iota-School/notebooks-for-all) team
+
+Draft 2: February 21, 2023
+
+---
+
+When writing & publishing Jupyter notebooks, the notebook editor has a huge influence on the content’s accessibility and there is not much an average user can do to fix that. Depending on the export method and content, notebooks can allow for different levels of accessibility for people using assistive technology. In running user tests, we’ve found that users encounter notebooks in a variety of formats:
+
+[The editable notebook format](https://jupyter.org/try-jupyter/lab?path=notebooks%2FIntro.ipynb) This is **inaccessible** and has many obstacles to navigating the UI or to edit cells
+
+[Uneditable notebooks exported to HTML](https://iota-school.github.io/notebooks-for-all/exports/Imaging_Sky_Background_Estimation-tab-to-content-nav-high-contrast.html) This is **somewhat accessible** because HTML is built for web accessibility. While there are issues, people tend to succeed in accessing a majority of exported notebooks.
+
+The content of notebooks. This **can be totally accessible**, if the authors are mindful of some of the following. This is where you can act now!
```

Comment:
If we want a link for this, here is an example of a notebook the Astronify team shared in HTML because we wanted it to be accessible: https://astronify.readthedocs.io/en/latest/astronify/Intro_Astronify_Series.html